core/src/main/resources/hudson/tasks/Maven/help-injectBuildVariables.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  Pass all build variables into maven process in form of java properties. This is seldom needed as Jenkins provides it as environment variables anyway.
```

Comment:
@jtnord, it works but the preferred way is to use the properties textarea (covered by `parametersReferencedFromPropertiesShouldRetainBackslashes` at least).

@KostyaSha, it is not a shame to contribute PR with nothing but tests, you know :wink:.